surface/include/pcl/surface/ball_pivoting.h

```diff
@@ -214,8 +305,10 @@ namespace pcl
      * @param num_point_in_radius
      * @param ratio_success
      */
-    virtual void
-    setEstimatedRadius (const int num_sample_point, const int num_point_in_radius, const float ratio_success);
+    void
+    setSearchRadiusAutomatically (const int num_sample_point = 0, 
```

Comment:
Can this be a typo? Before when the function was called `guess_radius` the default was 500.